applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle3N/ProjectParameters_stage2.json

```diff
@@ -0,0 +1,173 @@
+{
+  "problem_data": {
+    "problem_name"                     : "test_saturated_below_phreatic_level_pw_triangle3N_stage2",
+    "start_time"                       : 0.0,
+    "end_time"                         : 1.0,
+    "echo_level"                       : 1,
+    "parallel_type"                    : "OpenMP",
+    "number_of_threads"                : 1
+  },
+  "solver_settings": {
+    "solver_type"                      : "Pw",
+    "model_part_name"                  : "PorousDomain",
+    "domain_size"                      : 2,
+    "model_import_settings": {
+      "input_type"                     : "mdpa",
+      "input_filename"                 : "column"
+    },
+    "material_import_settings": {
+      "materials_filename"             : "../Common/MaterialParameters_stage2.json"
+    },
+    "time_stepping": {
+      "time_step"                      : 0.1,
+      "max_delta_time_factor"          : 1.0
+    },
+    "buffer_size"                      : 2,
+    "echo_level"                       : 1,
+    "clear_storage"                    : false,
+    "compute_reactions"                : true,
+    "reform_dofs_at_each_step"         : false,
+    "nodal_smoothing"                  : false,
+    "block_builder"                    : true,
+    "solution_type"                    : "Steady_State_Groundwater_Flow",
+    "scheme_type"                      : "Backward_Euler",
+    "reset_displacements"              : false,
+    "strategy_type"                    : "newton_raphson",
+    "convergence_criterion"            : "water_pressure_criterion",
+    "water_pressure_relative_tolerance": 0.0001,
+    "water_pressure_absolute_tolerance": 1E-09,
+    "displacement_relative_tolerance"  : 0.0001,
```

Comment:
This value is ten times smaller than what was used in stage 1. But there are no displacement DoFs here... So I would suggest to use the same value as in stage 1, since it is irrelevant anyway.

Reply:
To be consistent with other parts, in stage 1, it is changed to 0.0001